index.html

```diff
@@ -42,60 +42,67 @@
-                        Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. 
-                    </p>
-                    <p> If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-                    <button class="button">Start Collaboration</button>
+                        <p>
+                            Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. 
+                        </p>
+                        <p> If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                        <button class="button">Start Collaboration</button>
+                </div>
              </div>
+               
              <div>
                     <div class="geometry-image2">
                     </div>
                     <div class="geometry-image1">
                     </div>
               </div>
-         </div>     
+         
         </div>   
     </section>
+
+     <!--begining of project section -->
    <section class="grid-container">
             <section class="project-section">
-                <div>
-                    <h2 class="headings">Projects</h2>
+                <div class="headings a">
+                    <h2 >Projects</h2>
+                    <div class="a-img"></div>
                 </div>
         
-                <div class="project">
-                    <div class="laptop-img">
+                <div class="project b">
+                    <div class="laptop-img img">
                         <img src="images/laptop.png" alt="laptop">
                     </div>
-                    <div class="project-details">
-                        <div class="tech-details">
-                            <h2 class="project-name">Project name goes here</h2>
-                            <ul class="tech-stack">
+                    <div class="project-details  top">
+                        <div class="tech-details top-details">
+                            <h2 class="project-name head">Project name goes here </h2>
+                            <ul class="tech-stack tech">
                                 <li>HTML/CSS</li>
                                 <li>Ruby on Rails</li>
                                 <li>JavaScript</li>
                             </ul>
-                            <div class="details-button">
+                            <div class="details-button btn">
                                 <button>
                                     See this project
-                                    <div class="union"></div>
+                                    <div class="union white"></div>
+                                    <div class="union black"></div>
                                 </button>
                             </div>
                         </div>
                     </div>
```

Comment:
- Kindly, clear the space between images and keep the size and alignment of the Figma design to have better Ui 👍
### your Design 👇
![Screenshot 2023-05-12 214827](https://github.com/ctokoli/personal-portfolio-site/assets/52010789/af2d3288-cbc2-46cf-9712-bfdf5d121052)

### Figma design 👇
![Screenshot 2023-05-12 214654](https://github.com/ctokoli/personal-portfolio-site/assets/52010789/3215c80c-df63-4cf8-a624-9a49f6e6bfb9)